src/detect-pangram.js

```diff
@@ -15,7 +15,20 @@
  * @return {boolean}
  */
 function detectPangram(phrase) {
-  // write code here
+  const alphabet
+    = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n',
+      'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z'];
+  const alignText = phrase.toLowerCase();
+
+  if (phrase.length === 0) {
+    return false;
+  }
+  for (let i = 0; i < alphabet.length; i++) {
+    if (alignText.indexOf(alphabet[i]) === -1) {
```

Comment:
если нам не нужен индекс, то просто `includes`
```suggestion
    if (!phraseLowerCase.includes(alphabet[i])) {
```